pkg/logs/config/config.go

```diff
@@ -44,6 +26,7 @@ func buildLogSources(ddconfdPath string, collectAllLogsFromContainers bool, tcpF
 		sources = append(sources, containersSource)
```

Comment:
Could it be a problem that this source would be the first in the list?

Reply:
It's not anymore, I implemented a logic to assign the right source to the right container, see https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/input/docker/container.go#L63-L76

Reply:
👍